zerver/webhooks/github/tests.py

```diff
@@ -302,6 +302,21 @@ def test_check_run(self) -> None:
 """.strip()
         self.send_and_test_stream_message('check_run__completed', expected_topic, expected_message)
 
+    def test_team_edited_description(self) -> None:
+        expected_topic = "Team Testing"
+        expected_message = """The team description has been changed to: ```A temporary team so that I can get some webhook fixtures!```."""
```

Comment:
I think ideally it'd be "Fred changed the team description to:\n<blockquote>`.  I think you're missing a newline before the blockquote and should be using '``` quote'